packages/front-end/components/Modal.tsx

```diff
@@ -192,7 +194,7 @@ const Modal: FC<ModalProps> = ({
         </div>
       ) : (
         <>
-          {close && (
+          {close && showHeaderCloseButton && (
```

Comment:
We set header to null, and don't want to see a close button there at all according to Figma

Reply:
Kari mentioned that we will soon rework the upgradeModal overall with a new design that is closer to this one.  So that might become moot then.